lib/cbhandler.js

```diff
@@ -93,6 +93,12 @@ const handlePayload = (client, payload, signature, callbacks) => verifySignature
         .catch((err) => {
           console.log(err.message);
         });
+    } else if (action === 'targetTemperature') {
+      return ThermoStatTemp(payload, callbacks)
```

Comment:
is this suppose to be ThermoStatTemperature ?

Reply:
Nope, It can be anything. I just tried to follow the naming scheme